src/fileshelter/ui/FileShelterApplication.hpp

```diff
@@ -36,7 +36,8 @@ namespace UserInterface
 	{
 		public:
 			FileShelterApplication(const Wt::WEnvironment& env);
-
+            void updateMenuVisibility();
```

Comment:
I think you can make it private and call it from handlePathChange in FileShelterApplication.cpp
=> no need to call this from all the other places

Reply:
Better if that's possible - will check

Reply:
Done - simpler now, cheers